tools/txg/TXG.hs

```diff
@@ -422,7 +422,7 @@ poll version chainid = go
   where
     _ :<|> go :<|> _ :<|> _ = api version chainid
 
-listen :: ChainwebVersion -> ChainId -> ListenerRequest -> ClientM ApiResult
+listen :: ChainwebVersion -> ChainId -> ListenerRequest -> ClientM (CommandResult H.Hash)
```

Comment:
Do we have any coverage of TXG? @emmanueldenloye @fosskers

Reply:
Our tests don't test it, but perhaps they should be made to.